src/freertos_drivers/esp32/Esp32WiFiManager.cxx

```diff
@@ -1503,9 +1503,22 @@ StateFlowBase::Action Esp32WiFiManager::WiFiStackFlow::reload()
     {
         parent_->stop_uplink();
     }
+
+    if (parent_->connectionMode_ & CONN_MODE_SHUTDOWN_BIT)
+    {
+        // Alow a bit of time for lingering sockets to terminate.
+        return sleep_and_call(&timer_, MSEC_TO_NSEC(500), STATE(shutdown));
+    }
     return wait_and_call(STATE(reload));
 }
 
+StateFlowBase::Action Esp32WiFiManager::WiFiStackFlow::shutdown()
+{
+    LOG(INFO, "[WiFi] Shutting down.");
```

Comment:
How come we're not calling the gridconnect hub shutdown here? Where do we call that?